### PR TITLE
Update event filters naming and add sub label filter

### PR DESCRIPTION
**Pull request by NickM-27**
Here are some screenshots for context

Screen Shot 2022-05-05 at 10 14 07 PM

### CODE REVIEW

**Comment by blakeblackshear** [comment]
I'm hesitant to merge this one since I expect sublabels to be rarely used.

**Comment by NickM-27** [comment]
> I'm hesitant to merge this one since I expect sublabels to be rarely used. 

I can say the doubletake dev has agreed to add this automatically, but I do understand this isn't something users will be using as often and will mostly need services to support it which may be unlikely. 

A couple points:
1. I think it shouldn't be too difficult to make it so that the sub labels filter only shows if there is at least one that is set, would that make it better?
2. I think regardless, the updated names of the labels is something that's important

Either way, I totally understand your point and not a big deal if this is better left without 👍

**Comment by NickM-27** [comment]
@blakeblackshear Let me know if the above changes ease your concerns, if not I am good to go ahead and close this 👍 

In that case perhaps we can revisit in the future if requested or more popular than expected

**Comment by NickM-27** [comment]
Just as an update, double take 1.9.0 supports sub labels automatically: https://github.com/jakowenko/double-take/pull/209#issuecomment-1134164832

**Comment by jvrobert** [comment]
> I'm hesitant to merge this one since I expect sublabels to be rarely used.

I'm only one guy ;) but I think the double-take tagging is the killer feature that makes this worthwhile, personally. Thanks!

File: frigate/http.py [hunk 1]
```diff
@@ -251,6 +246,24 @@ def set_sub_label(id):
     )
 
 
+@bp.route("/sub_labels")
+def get_sub_labels():
+    try:
+        events = (
+            Event.select(
```
Comment (blakeblackshear) [inline frigate/http.py:253]: I think this should be `Event.select(Event.sub_label).distinct()`.

Comment (NickM-27) [reply frigate/http.py]: Yeah that makes more sense

**Pull request merged**